aws/sqs_event_manager/sqs_event_manager/app.py

```diff
@@ -177,3 +177,78 @@ def process_message(record: Dict) -> Dict[str, Union[str, bool]]:
         result['status'] = f'{type(error).__name__}: {error}'
         logger.error(result['status'])
     return result
+
+
+def subscription_success(message: AWSSNSMessage) -> Dict:
+    """
+    Notify SCC on subscription successfully processed
+    """
+    request_data = basic_request(message)
+    request_data['offerIdentifier'] = message.offer_id
+    request_data['isFreeTrialTermPresent'] = message.free_trial_term_present
+    return request_data
+
+
+def subscription_removed(message: AWSSNSMessage) -> Dict:
+    """
+    Notify SCC on subscription successfully removed
+    """
+    request_data = basic_request(message)
+    request_data['offerIdentifier'] = message.offer_id
+    request_data['isFreeTrialTermPresent'] = message.free_trial_term_present
+    return request_data
+
+
+def subscription_failed(message: AWSSNSMessage) -> Dict:
+    """
+    Notify SCC on subscription process failed
+    """
+    request_data = basic_request(message)
+    request_data['offerIdentifier'] = message.offer_id
+    request_data['isFreeTrialTermPresent'] = message.free_trial_term_present
+    return request_data
```

Comment:
`subscription_success`, `subscription_removed`,`subscription_failed` all do the same thing why repeat the code? Also the doc string doesn't match, neither of these functions actually notify SCC.

Reply:
done

Reply:
I already answered this to Sean. I want to keep the methods distinct as long as I don't have clear information from Alex that the request stays the same for all actions types. Once I know this, a refactor to reduce the duplication is easy and the preferred way as the other way round. So this stays and should not hurt that much

Reply:
I'm also 99% convinced that we have to add special sauce to this events sooner or later